nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2743,6 +2743,64 @@ nifi.content.repository.encryption.key.id=Key1
+
+Example: +
+`nifi.content.repository.hdfs.archive=archive1,archive2,archive3`
+|`nifi.content.repository.hdfs.operating.mode`|A comma separated list of operating modes that governs the behavior of the content repository. Default is `Normal`. +
+
+	The recognized modes and their behaviors are as follows: +
+
+	`Normal`: No special fallback handling is made during failure. Each configured location is written to as normal until they are full. Once all locations are full, writes will block until space becomes available. Note: This is default operating mode if one isn't specified in the `nifi.properties` file. +
+
+	`CapacityFallback`: The locations in the 'primary' group are filled first and the rest are only filled once all locations in the primary group are full. Once space becomes available again for at least a minute, the primary group will become active again. This mode cannot be used with the `FailureFallback` mode. The 'primary' group is specified with the following property where each location id is comma separated: `nifi.content.repository.hdfs.primary` +
+
+	`FailureFallback`: The configured locations 'primary' group are filled as normal until they are full. Once they are full, writes will block until space becomes available. If a write failure occurs within all primary locations, the remaining non-primary locations are written to until a configured time period has elapsed. This mode cannot be used with the `CapacityFallback` mode. +
+
+	`Archive`:  All locations are written to and filled as described in the other modes. As files are moved to the archive, they are copied to the locations in the 'archive' group and then deleted. This can be combined with any of the other three modes. If this is the only mode specified, `Normal` is also assumed. The 'archive' group is specified with the  `nifi.content.repository.hdfs.archive` property where each location id is comma separated.
+|`nifi.content.repository.hdfs.full.percentage`|The percentage ('##%') of a location's capacity that must be occupied before treating the location as 'full'. Note: Once a location is full, all writes will stop for that location. If all locations are full and there is no fallback, claim creation will stop until space becomes available. The default is `95%`.
+|`nifi.content.repository.hdfs.failure.timeout`|The amount of time to wait when a failure occurs for a location before attempting to use that location again for writing. Example value: `1 minute`
+|`nifi.content.repository.hdfs.wait.active.containers.timeout`|The amount of time to wait for an active location to be available before giving up and throwing an exception. Defaults to indefinite. Example value: `5 minutes`
+|`nifi.content.repository.hdfs.sections.per.container`|The number of subdirectories per location. Defaults to `1024`. This is primarily used to avoid too many content claim files within a single directory.
```

Comment:
claims go to single files (not directories) unless that is done differently here.

Reply:
Which property are you referring to? The claims operate the same way the default FileSystemRepository work: each physical resource claim file on disk will contain one or more content claims.